### DISCUSSION TITLE
Desktop app as frontend

### DISCUSSION THREAD

**Discussion by whois-pk** [realworld-apps/realworld#1143] [Q&A]
Though there doesn't exist a desktop app for Medium, how does it sound if we create a desktop app that replicates the web UI? I was planning to use [Compose Desktop](https://github.com/JetBrains/compose-jb) for doing so. 

Would a desktop client be useful for the realworld project?
Will replicating web UI for desktop be a good idea or some changes would have to be made to adapt to desktop?

**Comment by geromegrignon** [comment]
Hi @ pk-218, we may question the usefulness of a desktop application for such a project.
However, I consider a desktop version a valid example as the main goal of this project is to use the 'Medium' part as an excuse to showcase multiple technologies.

About the Web UI, it should be the same as defined in the specifications.

**Reply by whois-pk** [reply]
I will get on with the implementation then!

**Reply by geromegrignon** [reply]
As a side note, such an implementation is valid as a dedicated solution for desktop apps.
For wrappers like Electron, we would accept only one implementation as an example, not one for each existing web app implementation.
